src/mainwindow.cpp

```diff
@@ -10,14 +12,12 @@
 #include <QWebEngineFullScreenRequest>
 #include <QWebEngineProfile>
 #include <QWebEngineSettings>
-
 #include <QWebEngineUrlRequestInterceptor>
 #include <QWebEngineView>
 #include <QWidget>
 
 MainWindow::MainWindow(QWidget *parent)
-    : QMainWindow(parent), ui(new Ui::MainWindow)
-
+    : QMainWindow(parent), ui(new Ui::MainWindow), mpris(new NetflixMprisInterface)
```

Comment:
I Think we should have a default interface, so users of other services which do not currently have an interface can still get the basic functionality eg. play, pause, volume etc.

Reply:
Also if a user changes his provider how can we load a different interface if the mprisplayer has already been setup, might get amessge like this 
>MprisPlayer: Failed attempting to register object path. Already registered?

Tell me if I am talking nonsense

Reply:
> I Think we should have a default interface, so users of other services which do not currently have an interface can still get the basic functionality eg. play, pause, volume etc.
That's a good idea. I'm not sure, however, how that would work. Since the implementation of the MPRIS features is completely dependent on knowing the "correct" JavaScripts, there is no way to make one-size-fits-all implementation - what worked for Netflix will most definitely not work for say Amazon Prime or HBO Go.

Still, I believe I can provide a _dummy_ interface, which will serve as a fallback for these services, so that the JavaScripts will not cause errors in the console (video not found, play is not a function of `null`, etc.). Later, when all scripts are available, we can swap the dummy interface for the real thing and it will be a nice experience for the user.

> Also if a user changes his provider how can we load a different interface if the mprisplayer has already been setup.
Quite simply -- we just destroy the old one, create a new one a set it up again. :) I will add an example how to do this in the next commit.